### ISSUE TITLE
nsp's npm-shrinkwrap.json includes vulnerable module

### ISSUE THREAD

**Issue by reedloden** [nodesecurity/nsp#79]
https://github.com/nodesecurity/nsp/blob/master/npm-shrinkwrap.json includes a vulnerable version of `moment`. This causes `nsp` to complain about itself.

**Comment by evilpacket** [comment]
Thanks for pointing this out.

Notes:
This is due to the joi dependency.

**Comment by evilpacket** [comment]
we recently published nsp@2.2.2 which reports clean. I believe this should address this issue. Let us know if you still experience issues.

```
~/Documents/projects/nodesecurity/nsp (master)$ nsp check
(+) No known vulnerabilities found
```
